### ISSUE TITLE
A typo in README.md "parse new ."

### ISSUE THREAD

**Issue by oshliaer** [ParsePlatform/Anyimg#5]
The hint "Type `parse new .` in the directory where this README resides" is not work for me.
But it works in the `parse` directory
```console
Directory: C:\_\parse\Anyimg\Anyimg
Mode                LastWriteTime     Length Name
----                -------------     ------ ----
d----        21.01.2015     18:17            parse
-a---        21.01.2015     18:17         44 .gitignore
-a---        21.01.2015     18:17       1489 README.md

PS C:\_\parse\Anyimg\Anyimg> cd .\parse
PS C:\_\parse\Anyimg\Anyimg\parse> parse new .
Creating directory C:\_\parse\Anyimg\Anyimg\parse\.\config
Creating config file C:\_\parse\Anyimg\Anyimg\parse\.\config\global.json
Writing out sample file C:\_\parse\Anyimg\Anyimg\parse\.\public\index.html
Email: 
```

Cheers!

**Comment by huperniketes** [comment]
Indeed. This typo was a productivity killer for me, and I almost abandoned using any of the examples because of it.

Please either have the user execute it from the sub-directory "parse", or change the line creating the project to "parse new parse".

**Comment by html5cat** [comment]
Fixed by @huperniketes in 2eeb9a70f97996ed050d5124156ae38727aca426
Thanks!

**Comment by hramos** [comment]
Fixed in #6
